src/main.ts

```diff
@@ -50,13 +50,18 @@ async function main() {
 
 	const config: ISidecarConfig = configOrNull;
 
-	console.log(`Connecting to ${config.NAME} at ${config.WS_URL}`);
-
 	const api = await ApiPromise.create({
 		provider: new WsProvider(config.WS_URL),
 		types: config.CUSTOM_TYPES,
 	});
 
+	const chainName = await api.rpc.system.chain();
+	console.log(
+		`Connecting to ${chainName.toString()} via ${config.NAME} at ${
```

Comment:
At this point it's connected :)

Perhaps also change the `NAME` to be the client implementation name (e.g. [node-template](https://github.com/paritytech/substrate/blob/e824e8ab0fadec9949ebb8b9e14d98703d6b8d44/bin/node-template/runtime/src/lib.rs#L99) or [parity-polkadot](https://github.com/paritytech/polkadot/blob/edd8f3b22b36f850cece13a81aa8f6cb9f8532a3/runtime/polkadot/src/lib.rs#L92)). Then we get a message like, "Connected to Polkadot CC1 on the parity-polkadot client at ws://127.0.0.1:9944".
```suggestion
		`Connected to ${chainName.toString()} via ${config.NAME} at ${
```